com.unity.render-pipelines.high-definition/Editor/RenderPipeline/CustomPass/CustomPassVolumeEditor.cs

```diff
@@ -11,14 +11,15 @@
 
 namespace UnityEditor.Rendering.HighDefinition
 {
-    [CustomEditor(typeof(CustomPassVolume))]
+    [CustomEditor(typeof(CustomPassVolume)), CanEditMultipleObjects]
```

Comment:
If this become multi-editable, we must convert all location where we use `target` 
e.g.:
```
        void OnEnable()
        {
            m_Volume = target as CustomPassVolume;
```
in case of multiple selection we must work on each targets

Reply:
I removed the custom pass list, so all references to m_Volume are unused in multi-selection now.